htlcswitch/link.go

```diff
@@ -3996,72 +3997,6 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg) {
 				lnwire.ExperimentalEndorsementType,
 			)
 
-			switch fwdPkg.State {
```

Comment:
are you aware there's PR #10018?

Reply:
seems like 10018 could remove the `fwdFilter` replace it with a reforward boolean, because we do not really need it as a filter.